samples/ContinuousIntegration/README.md

```diff
@@ -0,0 +1,13 @@
+# ContinuousIntegration - React Native for Windows
+
+Whether you're building a new React Native Windows project or adding Windows support to an existing React Native project, at some point you may want to set up continuous integration.

```

Comment:
consider mentioning the value of CI in these cases, e.g.

```suggestion
Whether you're building a new React Native Windows project or adding Windows support to an existing React Native project, at some point you may want to set up continuous integration. This would ensure that your project and native module continue to work despite updates to either
- the app or module itself,
- an underlying dependency (e.g. react-native, react-native-windows), 
- the tool chain (different Visual Studio versions, compiler)
```